### PR TITLE
PTCaseStudy #1

### PR DESCRIPTION
**Pull request by lanceaune**
PLEASE DISREGARD FILE NAME 'yurts-e.html'. This should have been removed prior to adding.

### CODE REVIEW

**Comment by lanceaune** [comment]
last file submission is the correct one. Apologize for the many submissions ( i was running into a small error which was holding up the entire operation)

**Comment by mbMosman** [comment]
No worries on multiple submissions. Commit and push to GitHub as much as needed before the due date to get things correct.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </ul>
+        <div id="contact"><span class="resort">Pacific Trails Resort</span>
+            <br>12010 Pacific Trails Road
+            <br>Zephyr, CA 95555
+            <br>888-555-5555</div>
+        <br>
+        <footer>
+            <small><i>Copyright &copy; 2016 Pacific Trails Resort</i></small>
+            <br><small><i><a href="mailto:LanceAune@LanceAune.com">LanceAune@LanceAune.com</a></i></small></footer>
+    </main>
```
Comment (mbMosman) [inline index.html]: The main tag should only surround the main page content, this doesn't include the footer.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <li>Private yurts with decks overlooking the ocean</li>
+            <li>Activities lodge with fireplace and gift shop</li>
+            <li>Nightly fine dining at the overlook cafe</li>
+            <li>Heated outdoor pool and whirlpool</li>
+            <li>Guided hiking tours of the redwoods</li>
+        </ul>
+        <div id="contact"><span class="resort">Pacific Trails Resort</span>
+            <br>12010 Pacific Trails Road
+            <br>Zephyr, CA 95555
+            <br>888-555-5555</div>
```
Comment (mbMosman) [inline index.html]: You need a blank line between the address and phone.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <b><a href="index.html">Home</a>&nbsp; <a href="yurts.html">Yurts</a>&nbsp; Activities &nbsp; Reservations</b></nav>
+    <main>
+        <h2>Enjoy Nature in Luxury</h2>
+        <p><span class="resort">Pacific Trails Resort</span> offers a special lodging experience on the California North Coast. Relax in serenity with panoramic views of the Pacific Ocean.</p>
+        <ul>
+            <li>Private yurts with decks overlooking the ocean</li>
+            <li>Activities lodge with fireplace and gift shop</li>
+            <li>Nightly fine dining at the overlook cafe</li>
```
Comment (mbMosman) [inline index.html]: Typos in content will count against you so be careful with this. Imagine this is a site for a business owner. They'll care if things are spelled incorrectly, capitalization is off, and/or punctuation is bad.

Overlook Cafe should be capitalized.

File: yurts.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <b><a href="index.html">Home</a>&nbsp; <a href="yurts.html">Yurts</a>&nbsp; Activities &nbsp; Reservations</b></nav>
+    <main>
+        <h2>The Yurts at Pacific Trails</h2>
+        <dl>
+            <dt><strong>What is a yurt?</strong></dt>
+            <dd>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</dd>
+            <dt><strong>How are the yurts furnished?</strong></dt>
+            <dd>Each yurt is furnished with a queen size bed with down quilt and gas fired stove. The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</dd>
```
Comment (mbMosman) [inline yurts.html]: queen-sized with a dash.

File: yurts.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            <dt><strong>What should i bring?</strong></dt>
```
Comment (mbMosman) [inline yurts.html]: I should be capitalized

**Comment by mbMosman** [comment]
Overall good job. A few small things to fix, but nice work.

**Pull request closed without merging**